setup.py

```diff
@@ -159,6 +160,7 @@ def build_manpages():
         'scrub': ['mutagen>=1.33'],
         'bpd': ['PyGObject'],
         'replaygain': ['PyGObject'],
+        'reflink': ['reflink'],
```

Comment:
I don't think this is needed since it's not an optional dependency for a plugin

Reply:
Because it’s a conditional import (inside the relevant function instead of at the top of a module), I actually think it’s OK to be an optional dependency.

Reply:
so then should it be removed from the normal dependencies?

https://github.com/beetbox/beets/pull/3088/files/99cd7e2de457af1b05a2974fee975b48f15518dc#diff-2eeaed663bd0d25b7e608891384b7298R126

Reply:
Ah yes, good point.

Reply:
The line you refer to is in the `test` section, because it's required for some test cases, I think. Or am I looking at something else?

Reply:
Right again. Thanks, @rubdos.

Reply:
Ah okay, my mistake. In that case, maybe it should be noted in the docs that you need to install reflink? Either `python -m pip install beets[reflink]` or `python -m pip install reflink`

Reply:
Added in the config-docs!